src/sas/qtgui/Plotting/LinearFit.py

```diff
@@ -79,19 +78,22 @@ def __init__(self, parent=None,
         self.txtBerr.setText("0")
         self.txtChi2.setText("0")
 
-
         # Initial ranges
-        self.txtRangeMin.setText(str(max_range[0]))
-        self.txtRangeMax.setText(str(max_range[1]))
+        tr_min = GuiUtils.formatNumber(max_range[0])
+        tr_max = GuiUtils.formatNumber(max_range[1])
+        self.txtRangeMin.setText(str(tr_min))
```

Comment:
`GuiUitils.formatNumber` will only return a string. There is no need to cast it to str again